src/Core/Managed/Shared/Extensibility/Implementation/TaskTimerInternal.cs

```diff
@@ -72,7 +72,7 @@ public void Start(Func<Task> elapsed)
                     previousTask =>
 #endif
                     {
-                        CancelAndDispose(Interlocked.CompareExchange(ref this.tokenSource, null, newTokenSource));
+                        DisposeToken(Interlocked.CompareExchange(ref this.tokenSource, null, newTokenSource));
```

Comment:
Can you please update the comment for this method in `<summary>` task clarifying this behavior